docs/source/tutorials_source/platform/tutorial_aquarium_custom_metadata.py

```diff
@@ -0,0 +1,212 @@
+
+* How to upload custom metadata to your datasets.
+* How to detect and mitigate biases in your dataset through rebalancing.
+
+
+Requirements
+-------------
+
+You can use your own dataset or the one we provide for this tutorial. The dataset
+we will use is the training set of `Roboflow's Aquarium Dataset <https://public.roboflow.com/object-detection/aquarium>`_.
+It consists of 448 images from two aquariums in the United States and was made for object detection.
+You can download it here :download:`Aquarium.zip <../../../_data/Aquarium.zip>`.
+
+For this tutorial the `lightly` pip package needs to be installed:
+
+.. code-block:: bash
+
+    # Install lightly as a pip package
+    pip install lightly
+
+
+Custom Metadata
+-----------------
+
+In many datasets, images come with additional metainformation. For example, in autonomous driving,
+it's normal to record not only the image but also the current weather situation, the road type, and
+further situation specific information like the presence of vulnerable street users. In our case,
+the metadata can be extracted from the annotations.
+
+.. note:: 
+
+    Balancing a dataset by different available metadata is crucial for machine learning.
+
+Let's take a look at the dataset statistics from the `Roboflow website <https://public.roboflow.com/object-detection/aquarium>`_.
+The table below contains the category names and instance counts of the dataset
+
++---------------+----------------+
+| Category Name    | Instance Count |
++===============+================+
+| fish          | 2669           |
++---------------+----------------+
+| jellyfish     | 694            |
++---------------+----------------+
+| penguin       | 516            |
++---------------+----------------+
+| shark         | 354            |
++---------------+----------------+
+| puffin        | 284            |
++---------------+----------------+
+| stingray      | 184            |
++---------------+----------------+
+| starfish      | 116            |
++---------------+----------------+
+
+We can see that the dataset at hand is heavily imbalanced - the category distribution is long tail. 
+The most common category is `fish` and objects of that category appear 23 times more often than members 
+of the weakest represented category `starfish`. In order to counteract this imbalance, we want to 
+**remove redundant images which show a lot of fish** from the dataset.
+
+
+Let's start by figuring out how many objects of each category are on each image. The following code extracts metadata from COCO annotations and saves them in a 
+lightly-friendly format. If you want to skip this step, you can use the provided `_annotations.coco.metadata.json`.
+
+.. note:: You can save your own custom metadata with :py:class:`lightly.utils.io.save_custom_metadata`.
+
+.. code-block:: python
+
+    import json
+    from lightly.utils import save_custom_metadata
+
+    PATH_TO_COCO_ANNOTATIONS = 'PATH/TO/AQUARIUM/_annotations.cooo.json'
```

Comment:
Here there is `PATH/TO/AQUARIUM/`, but in line 134, there is `./aquarium`. This is inconsistent. The same applies also to the output file, perhaps this could be consistent. E.g. you could print out the path / filename in blue, like:
`print(f'Metadata is stored at: {bcolors.OKBLUE}{OUTPUT_FILE}{bcolors.ENDC}')`, same for the `PATH/TO/AQUARIUM/`